public/stylesheets/style.css

```diff
@@ -225,3 +226,7 @@ div[size="xl"] .modal-dialog {
 .login-container{
 	overflow: hidden;
 }
+#disabled{
```

Comment:
This one's used more than once and therefore should be a `class`. An `id` has to be unique and should only be used once.